aktuelt/serializers.py

```diff
@@ -1,5 +1,11 @@
 from rest_framework.fields import Field
 from wagtail.images.api.fields import ImageRenditionField
+from wagtail.rich_text import expand_db_html
+
+
+class NewsBodySerializer(Field):
+    def to_representation(self, value):
+        return expand_db_html(value)
```

Comment:
This is what Wagtail uses in own frontend when rendering this type of data, mainly via a `richtext` template filter